home_audit.ts

```diff
@@ -0,0 +1,691 @@
+const HesJsonSchema = require('./schema/hescore_json.schema.js');
+const Ajv = require("ajv");
+const addFormats = require('ajv-formats');
+const ajv = new Ajv({allErrors: true, strictTypes: false, strictSchema: false})
+addFormats(ajv);
+// Add the schema to the validator.
+ajv.addSchema(HesJsonSchema);
+// Add the custom keyword "error_msg" to the validator
+ajv.addKeyword('error_msg');
+
+const nullOrUndefined = [null, undefined];
+
+const mandatoryMessage = "Missing value for mandatory field";
```

Comment:
Probably leftover from legacy code, but I would prefer these use CAPITAL_SNAKE_CASE.

Reply:
Even better, `mandatoryMessage` is only ever referred to once, so I just made it a string literal in the one place we need it.

NULL_OR_UNDEFINED has been renamed :)